server/src/controllers/Products/IndexProductsController.js

```diff
@@ -9,7 +9,7 @@ class IndexProductsController {
     async index(req, res) {
         const { resetCode } = req.query;
 
-        const cartKeys = await this.redisClientService.eachScan('product:*');
+        const cartKeys = await this.redisClientService.eachScan('cart:*');
```

Comment:
@lukasz-bien-rmtcfm-com  I don't understand why we are fetched twice the same things ..

Reply:
it is a mistake, it should be `cart:*`, and it should be rather after this if statement below because it is now needed here after all

Reply:
it has been remade